campaigns.json

```diff
@@ -2537,8 +2530,7 @@
     "secondaries": {
       "/MinBias_TuneCUETP8M1_2017_5p02TeV-pythia8/RunIIpp5Spring18GS-94X_mc2017_realistic_v10For2017G_v3-v2/GEN-SIM": {
         "SiteWhitelist": [
-          "T2_US_MIT", 
-          "T2_DE_DESY"
+          "T2_US_MIT"
```

Comment:
Checked and when looking at RUCIO this was only at MIT and not at DESY so removed DESY